lib/Importer/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_definitions(-DGOOGLE_PROTOBUF_NO_RTTI)
 PROTOBUF_GENERATE_CPP(PROTO_SRCS PROTO_HDRS caffe.proto onnx.proto)
 
 add_library(Importer
```

Comment:
Can you add:

   target_compile_definitions(Importer
                              PRIVATE
                                -DGOOGLE_PROTOBUF_NO_RTTI)

Reply:
Done. Thanks! Note, that I can't remove `add_definitions(-DGOOGLE_PROTOBUF_NO_RTTI)` from this file.